Upstream-Status:Inappropriate [Backport]

Imported from perl git tree by Nitin A Kamble <nitin.a.kamble@intel.com>
2011-02-23

commit 24e93d7838b346d2ed632075f3d824a325170616
Author: Tony Cook <tony@develop-help.com>
Date:   Sat Aug 14 00:21:29 2010 +1000

    POSIX/t/posix.t expects a certain start to Makefile.PL
    
    6695a346 changed the start of Makefile.PL, but t/posix.t reads that to
    test its read() implementation, restore enough of the original for the
    test to pass.

diff --git a/ext/POSIX/Makefile.PL b/ext/POSIX/Makefile.PL
index 9e6d091..292882c 100644
--- a/ext/POSIX/Makefile.PL
+++ b/ext/POSIX/Makefile.PL
@@ -1,3 +1,4 @@
+# Expect this line to be read by t/posix.t, don't change it
 use ExtUtils::MakeMaker;
 use ExtUtils::Constant 0.11 'WriteConstants';
 use Config;
